gnovm/pkg/gnolang/garbage_collector.go

```diff
@@ -39,7 +39,9 @@ func (m *Machine) GarbageCollect() (left int64, ok bool) {
 	var visitCount int64
 
 	defer func() {
-		gasCPU := overflow.Mulp(visitCount*VisitCpuFactor, GasFactorCPU)
+		// Note: GasFactorCPU is only 1 at the moment, so multiplying any
+		// expression by this factor cannot overflow.
+		gasCPU := overflow.Mulp(visitCount, VisitCpuFactor) * GasFactorCPU
```

Comment:
maybe this even it is 1 for consistency and this factor can be changed
```suggestion
		gasCPU := overflow.Mulp(overflow.Mulp(visitCount, VisitCpuFactor), GasFactorCPU)
```

Reply:
That make sense, thank you for noticing